skaffold.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: skaffold/v1beta11
+apiVersion: skaffold/v2beta2
```

Comment:
this shouldn't be a problem since the skaffold version in lead-workspace and builder images has been updated.  we should probably also update this in springtrader too